src/Command/InstallCommand.php

```diff
@@ -188,8 +188,11 @@ private function addInitializePhp()
 /** @var Composer\Autoload\ClassLoader */
 $loader = require __DIR__ . '/../app/autoload.php';
 
+$request = Request::create('/_contao/initialize', 'GET', [], $_COOKIE, [], $_SERVER);
+$request->attributes->set('_scope', ('BE' === TL_MODE ? 'backend' : 'frontend'));
```

Comment:
Yes it is; in the custom entry point: https://github.com/isotope/core/blob/master/system/modules/isotope/group.php#L22